site-publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+rm -rf website-build
+npm run build-storybook
+npm run site-build
+
+pushd website-build
+
+git init
+# TODO: point this back at the upstream repo
+git remote add webpages git@github.com:tfennelly/jenkins-design-language.git
+git fetch --depth=1 webpages gh-pages
+
+git add --all
+git commit -m "webpages"
+git merge --no-edit -s ours remotes/webpages/gh-pages
+
+git push webpages master:gh-pages
+
+rm -rf .git
```

Comment:
No, `website-build` is in `.gitignore` and the site is generated into it i.e. the generated stuff is never committed to the repo. We just set up a temp remote onto the repo inside this dir and push the generated website to the `gh-pages` branch and then remove the remote.